### ISSUE TITLE
[Suggestion] "Allow Spawn as NPC" option for player builds

### ISSUE THREAD

**Issue by Asnro** [tsunamayo/Starship-EVO#5314]
Currently the selection of NPC ships and stations is rather limited, and downloading creations from workshop or building your own stuff feels without purpose as only the player(s) can interact with them.

I propose an option to insert the player's own creations into the NPC spawn list, for several reasons:
 - Diversify the number of NPCs you can see flying around
 - See your own builds as part of the game's environment
 - Give Creative mode a greater purpose (Creating ships for NPCs instead of simply designing blueprints for yourself)
 - Give the option of more difficult pirates, bounty targets, and police
 - Download creations from workshop not just to fly around in, but to diversify your own galaxy

I envision the option as a simple dropdown menu, with the options "None", "Civilian", "Pirate", or "Police"

**Comment by ProPeach** [comment]
You can actually do this currently by placing the bp of your ship in the correct folder in the game files, I believe they're in the Steam folder rather than the Appdata/bp file route

**Comment by Asnro** [comment]
> You can actually do this currently by placing the bp of your ship in the correct folder in the game files. I cant remember the exact folder atm, but I believe they're in the Steam folder rather than the Appdata/bp file route

Does this work for stations? I took a look at the NPC ships folder and saw for example the AAM station folder which is completely disabled from spawning in-game, which makes me think stations have a more complex spawning system.

**Comment by tsunamayo** [comment]
Yes you should already be able to do that, I will work on a more user friendly solution, to add or even replace the base blueprint. Station generation is complicated but should be doable, and yes I have plenty of blueprint to add including another station set.
I still have to do more work on factions though.
Thanks
